api/server/routes/oauth.js

```diff
@@ -14,6 +14,21 @@ const domains = {
 
 router.use(loginLimiter);
 
+const authenticate = (strategy, options) => {
+  return (req, res, next) =>
+    passport.authenticate(strategy, options, (err, _user, info) => {
+      if (err) {
+        return next(err);
+      }
+
+      if (info) {
```

Comment:
Thanks again for the PR. Attempting login without the environment variables set, it looks like `info` is always defined and returns with redirect and the empty object: 
![image](https://github.com/danny-avila/LibreChat/assets/110412045/9c8d1a86-a42b-4ca1-8db7-3e6622e93435)

Tested with google, github, and discord, all would include an empty info object which generates the `[object Object]` string.

You could handle the edge case of the object string, as well as make sure this condition only happens when it's truly an error.

Reply:
I've tested the case of an existing user logging in now, I would appreciate it if you check it again.